cves/CVE-2008-2364.yml

```diff
@@ -31,7 +31,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  A flaw was discovered when a server would have to handle excessive interim responses. In a situation where a user who is using the proxy is tricked into visiting a 
+  malicious web-server, the proxy would need to use stack or heap memory. If this was continually done
```

Comment:
I'm also wondering about this line. I would imagine that the proxy would _always_ need to use some amount of stack or heap memory. Is this issue related to excessive/unbounded memory use?